app/build.gradle

```diff
@@ -32,7 +32,7 @@ android {
         applicationId "ai.susi"
         minSdkVersion rootConfiguration.minSdkVersion
         targetSdkVersion rootConfiguration.targetSdkVersion
-        versionCode 7
+        versionCode 8
         versionName "1.0.6"
```

Comment:
Change

Reply:
@iamareebjamal Do I need to change the versionName? The version is 1.0.6 that is why I did not change it.

Reply:
Yes both versionCode and versionName need to change with each dev cycle

Reply:
@iamareebjamal The latest version on play store is 1.0.6. The version name in the file was already 1.0.6, therefore I did not change it. If it needs to change, it would be great if you could please let me know what should i name it as.

<img width="435" alt="screen shot 2018-05-25 at 10 24 33 pm" src="https://user-images.githubusercontent.com/30979369/40556488-734279bc-606a-11e8-929a-eca3e9cd7289.png">

Reply:
versions are changed **before** the app is uploaded to Play Store. If you check the commit history, I already changed 5 -6 and 1.0.6, so naturally for next development cycle, this needs to change to 1.0.7

Reply:
@iamareebjamal Thanks for the clarification. :)
I will do that asap.

Reply:
@iamareebjamal This has been fixed already. Would you please have a look?